runtime/src/accounts_index.rs

```diff
@@ -831,12 +834,14 @@ impl<T: 'static + Clone> AccountsIndex<T> {
         self.purge_secondary_indexes_by_inner_key(pubkey, Some(&purged_slots), account_indexes);
     }
 
+    // `is_cached` closure is needed to work around the generic (`T`) indexed type.
```

Comment:
Alternative to the is_cached would be just add another Trait bound to `AccountsIndex` like this:

```rust
impl<T: 'static + Clone + IsCached > AccountsIndex<T>
impl IsCached for AccontInfo {
    fn is_cached() -> { ... }
}
```

In my opinion, Accountsindex is too generic. I think it's ok to be more bound. :)

Reply:
Yeah, I like this, done!

Reply:
nicely done! Then I think this comment itself serves no purpose now. :)